CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.3.6
```

Comment:
Can you please omit the changes to the `CHANGELOG`? I usually separate them out into the tagged commits during the release process. I also think we'll bump to 0.4.0 because of the jump from 9.4 to 9.5.